docs/ros-bridge.md

```diff
@@ -19,21 +19,18 @@ roslaunch fsds_ros_bridge fsds_ros_bridge.launch
 ```
 
 ## Publishers
-- `/fsds_ros_bridge/origin_geo_point` [fsds_ros_bridge/GPSYaw](../ros/src/fsds_ros_bridge/msg/GPSYaw.msg)   
-GPS coordinates corresponding to global NED frame. This is set in the airsim's [settings.json](https://microsoft.github.io/AirSim/docs/settings/) file under (located [here](../../../UE4Project/Plugins/AirSim/Settings/settings.json)) the `OriginGeopoint` key. 
-
 - `/fsds_ros_bridge/VEHICLE_NAME/global_gps` [sensor_msgs/NavSatFix](https://docs.ros.org/api/sensor_msgs/html/msg/NavSatFix.html)   
 This the current GPS coordinates of the drone in airsim. 
 
-- `/fsds_ros_bridge/VEHICLE_NAME/odom_local_ned` [nav_msgs/Odometry](https://docs.ros.org/api/nav_msgs/html/msg/Odometry.html)   
-Odometry in NED frame wrt starting point.  THIS WILL NOT BE STREAMED DURING COMPETITION.
+- `/fsds_ros_bridge/VEHICLE_NAME/odom` [nav_msgs/Odometry](https://docs.ros.org/api/nav_msgs/html/msg/Odometry.html)
+Ground truth car position and orientation in NED frame. THIS WILL NOT BE STREAMED DURING COMPETITION.
 
 - `/fsds_ros_bridge/VEHICLE_NAME/CAMERA_NAME/IMAGE_TYPE/camera_info` [sensor_msgs/CameraInfo](https://docs.ros.org/api/sensor_msgs/html/msg/CameraInfo.html)
```

Comment:
do we want to keep the fsds_ros_bridge namespace or make it /fsds?

Reply:
These settings are inside the `fsds_ros_bridge` node, so they are placed in the `fsds_ros_bridge` namespace automatically.

Reply:
No, the node name name in the launchfile determains the namespace of the parameters.